common/constants.go

```diff
@@ -305,8 +305,9 @@ const (
 )
 
 const (
-	ConfigStemApp      = "edgex/appservices/"
-	ConfigStemCore     = "edgex/core/"
-	ConfigStemDevice   = "edgex/devices/"
-	ConfigStemSecurity = "edgex/security/"
+	ConfigStemAll      = "edgex/3.0"
```

Comment:
propose to use **v3** instead of **3.0**, because we normally don't upgrade it in 3.1 or 3.2.
However, we can upgrade to v3.1 or v3.2 if necessary.  **3.0** is inaccurate.